src/components/DataTable/DataTableRow/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { flexRender } from "@tanstack/react-table";
```

Comment:
Recently, we have decided to not use index.tsx for component file names. Please modify accordingly in all places.